frontend/src/tests/modules/TestStatusRedux.test.js

```diff
@@ -0,0 +1,15 @@
+import testStatus, {
+  activateTest,
+  initialState,
+  deactivateTest
+} from "../../modules/TestStatusRedux";
+
+describe("setLanguage action", () => {
```

Comment:
Typically you'd have a describe block per functional area of, so here it'd be: `describe("test activation state", () => {`

Then you'd have an it block within that for each type of thing you are testing. So I'd split your test into two `it` blocks, one for `activateTest` and one for `deactivateTest`. This separation makes it a bit easier to tell exactly what fails or passes.

Reply:
fixed in 9ee301c